eva-submission-docker/eva_docker/Dockerfile

```diff
@@ -0,0 +1,80 @@
+FROM ubuntu:22.04
+
+ENV JAVA_VERSION 8
+ENV NXF_VER 21.10.5
+ENV VCF_VALIDATOR_VERSION 0.9.4
+ENV MONGO_VERSION 4.4
+ENV EVA_ACCESSION_VERSION v0.6.21
+ENV EVA_PIPELINE_VERSION v2.9.1
+ENV EVA_SUBMISSION_VERSION v1.10.6
+
+COPY maven-settings.xml /root
+COPY eva_docker/submission_config.yml /root/.submission_config.yml
+
+RUN mkdir -p /usr/local/test_eva_submission/data
+RUN mkdir -p /usr/local/test_eva_submission/submissions
+RUN mkdir -p /usr/local/eva/public_ftp/
+RUN mkdir -p /usr/local/eva/datasources/reference_sequences
+RUN mkdir -p /usr/local/eva/datasources/vep-cache
+RUN mkdir -p /usr/local/software/opencga
+
+ADD eva_docker/ftp /usr/local/eva/ftp/
+ADD eva_docker/conf /usr/local/software/opencga/conf
+
+# install utility libs
+RUN apt update  \
+    && apt install -y curl wget maven git vim samtools bcftools tabix bedtools minimap2 rsync  \
```

Comment:
This is probably fine for testing but I would prefer installing samtools, bcftools, tabix, bedtools and minimap2 from source to controls which version we're installing. 
That's very important if we're running this in production but I guess it should do for now.